ddl/backfilling_dispatcher.go

```diff
@@ -489,6 +493,25 @@ func getSummaryFromLastStep(
 	return minKey, maxKey, totalKVSize, allDataFiles, allStatFiles, nil
 }
 
+func cleanupCloudStorageFiles(ctx context.Context, gTaskMeta *BackfillGlobalMeta) {
+	backend, err := storage.ParseBackend(gTaskMeta.CloudStorageURI, nil)
+	if err != nil {
+		logutil.Logger(ctx).Warn("cannot cleanup cloud storage files", zap.Error(err))
+		return
+	}
+	extStore, err := storage.New(ctx, backend, &storage.ExternalStorageOptions{})
+	if err != nil {
+		logutil.Logger(ctx).Warn("cannot cleanup cloud storage files", zap.Error(err))
+		return
+	}
+	prefix := strconv.Itoa(int(gTaskMeta.Job.ID))
+	err = external.CleanUpFiles(ctx, extStore, prefix, cleanExternalStoreFilesConcurrency)
```

Comment:
maybe add a integration test for it, see https://github.com/D3Hunter/tidb/blob/2c722e80c21971cd354ce9a774ea017755df29b1/tests/realtikvtest/importintotest4/global_sort_test.go#L59